en/docs/concepts/authentication/multi-factor-authentication.md

```diff
@@ -0,0 +1,46 @@
+# Multi-Factor Authentication
+
+## What is multi-factor authentication?
+
+Multi-factor authentication (MFA) is an effective method of providing enhanced security during user authentication by verifying user identity using more than one piece of identifying information. This means that in addition to providing basic authentication (username and password credentials), the user is also prompted to provide further verfication in the form of a one-time password, biometrics, hardware key devices etc. 
+
+Due to increasing digital crimes and internet fraud in the world, people are highly concerned about security. The traditional username and password method is not enough to secure the authentication. Processing speeds of CPUs have increased, so brute force attacks are a reality and dictionary attacks have become a common threat. GPGPU password cracking and rainbow tables have provided similar advantages to attackers.
+
+Thereby, MFA creates a layered defense and makes it more difficult for an unauthorized person to gain access. Even if the username and password is compromised, the attacker would still have atleast one more barrier to break through in order to successfully breach security. 
+
+---
+
+## Authetication factors
+
+Multi-factor authentication uses a combination of different types of authentication factors. The three main categories are: 
+
+- Knowledge factors: Things only the user knows, such as passwords or PIN numbers.
+
+- Possession factors: Things only the user has, such as ATM cards or hardware key devices.
+
+- Inherence factors: Things only the user is, such as a fingerprint or face.
+
+A common day-to-day example of using a combination of these factors for MFA can be seen when withdrawing money from a bank. In this case, the ATM card is the possession factor and the PIN number is the knowledge factor.
+
+---
+
+## Using multi-factor authentication
+
+By default, WSO2 Identity Server (WSO2 IS) is shipped with username-and-password-based authentication along with support for adding additional steps of authentication with authenticators such as SMSOTP, FIDO, MEPin, etc. WSO2 Identity Server allows configuring multi-step authentication that enables defining an authentication chain containing different authenticators in different steps. This chain can also be converted to a multi-factor authentication chain by adding different factors of authentication as options to each step. 
+
+!!! tip
+    For a complete list of readily available authenticators, see the [WSO2 IS Connector Store](https://store.wso2.com/store/assets/isconnector/list). 
+
+Note that there are certain second-factor authenticators that **must** be followed by a knowledge factor authenticator. For instance, when you configure the FIDO authenticator, it must not be configured as the first step and must be followed by a first factor authenticator first.
```

Comment:
Do we need to mention the reason for this?

Reply:
it might be a bit complicated to explain why. Its just the way its implemented and is usually because things like biometric authenticators are not sufficient enough as a first factor yet.

Reply:
I just felt that the **must** condition without an explanation might make the feature look inflexible. If we could at least make sure that it's an industry standard way or a known issue, and mention the same it might be more convincing. Don't you think?